examples/restrict-by-media-type/restrict_by_media_type.client.out

```diff
@@ -18,8 +18,8 @@ $ curl -v http://localhost:9092/infoService -H "Accept:application/xml" -H "Cont
 <
 * Connection #0 to host localhost left intact
 
-# Use the following client to invoke the service with a media type that is not acceptable. The media type mentioned
-# in the Accept header is not listed under produces resource configuration.
+# To invoke the service with a media type that is not acceptable, execute the following client. The media type mentioned
```

Comment:
The word "execute" is more relevant than "use" here since the context is about a cURL command. 

Changed to: "To invoke the service with a media type that is not acceptable, execute the following cURL request.".